src/index.js

```diff
@@ -0,0 +1,37 @@
+import './style.css';
+
+const todoList = [];
+let list = [];
```

Comment:
* According to the requirements you are expected to 
* Create an index.js file and set an array of some simple to-do tasks (array of objects).
* Write a function to iterate over the tasks array and populate an HTML list item element for each task
* On page load render the dynamically created list of tasks in the dedicated placeholder. The list should appear in order of the index values for each task.
* Kindly make the necessary adjustments 
![image](https://github.com/faisalakb/Todo_List/assets/52497246/493f5944-ee12-49ca-b934-c26915ad0a50)